src/dotnet-interactive-vscode-insiders/README.md

```diff
@@ -2,31 +2,83 @@ This extension is currently **_in preview_**.
 _The Visual Studio Code notebook support that this extension uses is also in preview and design is ongoing, so the extension might not work._
 ---
 
-# .NET Interactive Notebooks
+# Polyglot Notebooks
```

Comment:
Since this is where the majority of our users are, we might want to hoist this section closer to the top of the page.